bridge/hardhat.config.js

```diff
@@ -155,54 +197,56 @@ function getNamedAccounts() {
 
 function getMultiSigAddressesByChainId() {
   let multiSigAddressesByChainId = {};
-  multiSigAddressesByChainId[ETHEREUM_MAIN_NET_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
-  multiSigAddressesByChainId[RSK_MAIN_NET_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
-  multiSigAddressesByChainId[RSK_TEST_NET_CHAIN_ID] = '0x88f6b2bc66f4c31a3669b9b1359524abf79cfc4a';
-  multiSigAddressesByChainId[ETHEREUM_KOVAN_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
-  return multiSigAddressesByChainId;
+  multiSigAddressesByChainId[chains.ETHEREUM_MAIN_NET_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
+  multiSigAddressesByChainId[chains.RSK_MAIN_NET_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
+  multiSigAddressesByChainId[chains.RSK_TEST_NET_CHAIN_ID] = '0x88f6b2bc66f4c31a3669b9b1359524abf79cfc4a';
+  multiSigAddressesByChainId[chains.KOVAN_TEST_NET_CHAIN_ID] = '0x040007b1804ad78a97f541bebed377dcb60e4138';
+  return chains.multiSigAddressesByChainId;
```

Comment:
hey, I think all of these returns (for each `getBlaBlaAddressesByChainId`) are broken, right? we should return the dictionary we just built

Reply:
broken like ?

Reply:
got it, thanks man

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/269/commits/d508f5745a007b718e479e6ecc14ce0e6e7a87ea